frontend/modules/dialog/dialog.redux.ts

```diff
@@ -161,6 +164,18 @@ const showLoggedOutDialog = (state = INITIAL_STATE, action) => {
 	return showDialog(state, { config });
 };
 
+const showUnauthorizedModelAccessDialog = (state = INITIAL_STATE, action) => {
+	const config = {
+		title: 'Unauthorized Access',
+		content: 'Insufficient permission to access the model',
+		template: Dialogs.UnauthorizedAccessDialog,
+		onCancel: () => push(ROUTES.TEAMSPACES),
+		onConfirm: () => {}
```

Comment:
should it not also do push(ROUTES.TEAMSPACES) here? so we don't have the redirect logic on both this place and the component?

Reply:
Yes, You are right, it makes a lot more sense, thanks for the suggestion.